packages/math-input/src/components/keypad/keypad.tsx

```diff
@@ -71,7 +72,8 @@ export default function Keypad(props: Props) {
         React.useState<TabbarItemType>("Numbers");
     const [isMounted, setIsMounted] = React.useState<boolean>(false);
 
-    const availablePages = allPages(props);
+    // We don't want any tabs available on mobile fractions keypad
+    const availablePages = props.fractionsOnly ? [] : allPages(props);
```

Comment:
I think this is fine to keep, but I'm wondering if this is also unneeded code.

Pages are determined by certain flags in the keypad config (like `advancedRelations`) which _I think_ are all going to be falsy in the components that use the fraction keypad.

So I say keep it, but I'm also going to add this note for any future gumshoes investigating why we put this here (and maybe it's worth noting in the comment?)

Reply:
Hmm I think you have a good point here. The issue is that our current logic for allPages is that we always return _at least_ the Numbers page. I do have a proposed fix however that I can implement that I think will help clarify this situation for ourselves. It looks like we're only using this method and variable to specifically create the list of tabs, so I can adjust it to be a little more specific. 

- I'll rename the variable "availablePages" to "availableTabs"
- I'll rename the method "allPages" to "getAvailableTabs" 
- I'll move the logic to return an empty array to "getAvailableTabs" so that all the logic is contained in one place.